sdk/healthdataaiservices/azure-health-deidentification/customization/src/main/java/HealthDeidentificationSdkCustomization.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import com.azure.autorest.customization.*;
+import org.slf4j.Logger;
+
+public class HealthDeidentificationSdkCustomization extends Customization {
+    @Override
+    public void customize(LibraryCustomization customization, Logger logger) {
+        // Do it manually by editing DeidServicesAsyncClient class
```

Comment:
Do we want to remove this?

Reply:
Yes this is not needed anymore. I wanted to keep it as a placeholder but I can remove it